### PR TITLE
AP_Frsky_Telem: readability and declaration improvements

### PR DESCRIPTION
**Pull request by floaledm**
As suggested by tridge:
- added extra parentheses around pointed to values for readability
- added const to some declarations

### CODE REVIEW

**Comment by magicrub** [comment]
Travis build is failing:

> /home/travis/build/ArduPilot/ardupilot/libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp: In constructor 'AP_Frsky_Telem::AP_Frsky_Telem(AP_AHRS&, const AP_BattMonitor&, const RangeFinder&)':
> /home/travis/build/ArduPilot/ardupilot/libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp:28:1: error: uninitialized const member in 'const enum AP_SerialManager::SerialProtocol' [-fpermissive]
>  AP_Frsky_Telem::AP_Frsky_Telem(AP_AHRS &ahrs, const AP_BattMonitor &battery, const RangeFinder &rng) :
>  ^
> compilation terminated due to -Wfatal-errors.
>

File: libraries/AP_Frsky_Telem/AP_Frsky_Telem.h [hunk 1]
```diff
@@ -142,23 +142,23 @@ class AP_Frsky_Telem
     AP_AHRS &_ahrs;
     const AP_BattMonitor &_battery;
     const RangeFinder &_rng;
-    AP_HAL::UARTDriver *_port;                  // UART used to send data to FrSky receiver
-    AP_SerialManager::SerialProtocol _protocol; // protocol used - detected using SerialManager's SERIAL#_PROTOCOL parameter
-    bool _initialised_uart;
+    const AP_HAL::UARTDriver *_port;                  // UART used to send data to FrSky receiver
+    const AP_SerialManager::SerialProtocol _protocol; // protocol used - detected using SerialManager's SERIAL#_PROTOCOL parameter
```
Comment (khancyr) [inline libraries/AP_Frsky_Telem/AP_Frsky_Telem.h]: this couldn't be const as you assign it later

**Comment by floaledm** [comment]
Thanks for your review. I made the changes...

**Comment by OXINARF** [comment]
@floaledm Any reason why this needs to go to 3.4.2? This is basically a code improvement, but I'm not seeing any new feature nor bug fixing.

**Comment by magicrub** [comment]
I changed milestone to v3.5.0 since this is non-critical for the 3.4.x branch.

**Comment by floaledm** [comment]
Thanks Tom!

**Pull request merged**